testutil/port.go

```diff
@@ -0,0 +1,35 @@
+package testutil
+
+import (
+	"math/rand"
```

Comment:
Wonder if we should use crypto rand here instead?

Reply:
Would it be enough to seed the PRNG with the current time?

Reply:
That'd be fine. If done, however, it should be instantiated in `testutil` package so that we don't introduce conflicts if e.g. a command has its own seed, etc. (I.e. don't change global state.)